sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTblPropertiesSuite.scala

```diff
@@ -56,7 +44,7 @@ trait ShowTblPropertiesSuiteBase extends command.ShowTblPropertiesSuiteBase
     }
   }
 
-  test("SHOW TBLPROPERTIES FOR TEMPORARY IEW") {
+  testV1("SHOW TBLPROPERTIES FOR TEMPORARY VIEW") {
```

Comment:
are you saying, by default `SHOW TBLPROPERTIES` can't work with temp view? That's a terrible breaking change and we should fix it.

Reply:
ok. I try do it.

Reply:
A simple idea is to force to use v1 command if the target is a view